shard.lock

```diff
@@ -73,13 +73,17 @@ shards:
     version: 2.2.0
 
   rethinkdb:
-    git: https://github.com/place-labs/crystal-rethinkdb.git
+    git: https://github.com/aca-labs/crystal-rethinkdb.git
```

Comment:
not sure why this has reverted back to `aca-labs`?

Reply:
neither.

Reply:
try a `shards list --tree`